lib/bootloader_pvm.pm

```diff
@@ -123,19 +123,24 @@ sub enter_netboot_parameters {
         record_info("Updated boot path for PPC64LE_GRUB_HTTP defined", $mntpoint);
     }
     my $ntlm_p = get_var('NTLM_AUTH_INSTALL') ? $ntlm_auth::ntlm_proxy : '';
-    if (get_var('AGAMA')) {
-        type_string_slow "linux $mntpoint/linux vga=normal root=live:http://" . get_var('OPENQA_HOSTNAME') . "/assets/iso/" . get_var('ISO');
-        type_string_slow " " . get_var('EXTRABOOTPARAMS');
+    if (is_agama) {
```

Comment:
This may lead to issue, AGAMA we used as non-unattended test, AGAMA_AUTO used for unattended test.

Reply:
Good comment, but don't worry, it will not introduce the new issue,  the `agama.auto` shows up only we define AGAMA_AUTO parameter.  please see more info at https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20777#discussion_r1881708406